### ISSUE TITLE
SIP container error during request processing

### ISSUE THREAD

**Issue by xjarkrof** [OpenLiberty/open-liberty#14367]
**Bug Description**
When trying to deploy SIP application the sip container is unable to process any request with following error in the log:
```
[10/7/20, 1:46:14:472 CEST] 00000044 com.ibm.ws.sip.container.was.message.SipMessage              E CWSCT0139E: SIP Failed to invoke request; message is [Ljava.lang.Object;@XXXXXX
```
**Steps to Reproduce**
Create simple SIP application with ```SipServlet``` override any ```doXXXX``` method.

**Expected behavior**
The override method should be called

**Diagnostic information:**
 - OpenLiberty Version: _20.0.0.10_
 - Java Version: 
```
openjdk version "11.0.8" 2020-07-14
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.8+10)
Eclipse OpenJ9 VM AdoptOpenJDK (build openj9-0.21.0, JRE 11 Linux amd64-64-Bit Compressed References 20200715_697 (JIT enabled, AOT enabled)
OpenJ9   - 34cf4c075
OMR      - 113e54219
JCL      - 95bb504fbb based on jdk-11.0.8+10)
```
 - server.xml
```
<server>
    <featureManager>
        <feature>jaxrs-2.1</feature>
        <feature>cdi-2.0</feature>
        <feature>sipServlet-1.1</feature>
        <feature>jndi-1.0</feature>
        <feature>jsonp-1.1</feature>
        <feature>jaxrsClient-2.1</feature>
        <feature>servlet-4.0</feature>
        <feature>localconnector-1.0</feature>
        <feature>jaxws-2.2</feature>
    </featureManager>
    <variable name="default.http.port" defaultValue="9080"/>
    <variable name="default.https.port" defaultValue="9443"/>
    <sipContainer invalidateSessionOnShutdown="true" msgArrivalTimeAttr="true" markInternalResponse="true" />
    <sipEndpoint host="localhost" sipTCPPort="5060" bindRetries="60" bindRetriesDelay="5000" />
    <httpEndpoint host="*" httpPort="${default.http.port}" httpsPort="${default.https.port}" id="defaultHttpEndpoint"/>
</server>
```
 - Exception Info: 
**exception summary**
```
 Index  Count  Time of first Occurrence    Time of last Occurrence     Exception SourceId ProbeId
------+------+---------------------------+---------------------------+---------------------------
     0     30   10/7/20, 1:46:13:150 CEST   10/7/20, 1:46:14:472 CEST java.lang.ClassCastException com.ibm.ws.webcontainer.internal.WebContainer 162
```
**ffdc log**
```
------Start of DE processing------ = [10/7/20, 1:46:13:154 CEST]
Exception = java.lang.ClassCastException
Source = com.ibm.ws.webcontainer.internal.WebContainer
probeid = 162
Stack Dump = java.lang.ClassCastException: com.ibm.ws.sip.container.was.message.SipMessage incompatible with com.ibm.websphere.servlet31.request.IRequest31
	at com.ibm.ws.webcontainer31.srt.SRTServletRequest31.getContentLengthLong(SRTServletRequest31.java:217)
	at com.ibm.ws.webcontainer31.srt.SRTServletRequest31.initForNextRequest(SRTServletRequest31.java:127)
	at com.ibm.ws.webcontainer40.srt.SRTServletRequest40.initForNextRequest(SRTServletRequest40.java:97)
	at com.ibm.ws.webcontainer.srt.SRTConnectionContext.prepareForNextConnection(SRTConnectionContext.java:66)
	at com.ibm.ws.webcontainer.WebContainer.handleRequest(WebContainer.java:783)
	at com.ibm.ws.webcontainer.osgi.DynamicVirtualHost$2.run(DynamicVirtualHost.java:279)
	at com.ibm.ws.sip.container.was.message.SipMessage.dispatch(SipMessage.java:1435)
	at com.ibm.ws.sip.container.was.message.SipMessage.run(SipMessage.java:1390)
	at com.ibm.ws.sip.container.was.WebsphereInvoker.invokeSipServlet(WebsphereInvoker.java:142)
	at com.ibm.ws.sip.container.router.SipRouter.invokeSipServlet(SipRouter.java:1740)
	at com.ibm.ws.sip.container.tu.TransactionUserImpl.handleUASRequest(TransactionUserImpl.java:1798)
	at com.ibm.ws.sip.container.tu.TransactionUserImpl.processRequestUASMode(TransactionUserImpl.java:1891)
	at com.ibm.ws.sip.container.tu.TransactionUserImpl.processRequest(TransactionUserImpl.java:1623)
	at com.ibm.ws.sip.container.tu.TransactionUserWrapper.processRequest(TransactionUserWrapper.java:910)
	at com.ibm.ws.sip.container.transaction.ServerTransaction.processRequest(ServerTransaction.java:98)
	at com.ibm.ws.sip.container.router.tasks.RequestRoutedTask.doTask(RequestRoutedTask.java:93)
	at com.ibm.ws.sip.container.router.tasks.InitialRequestRoutedTask.doTask(InitialRequestRoutedTask.java:58)
	at com.ibm.ws.sip.container.router.tasks.RoutedTask.run(RoutedTask.java:92)
	at com.ibm.ws.sip.container.was.SignalingEndOfTask.run(SignalingEndOfTask.java:76)
	at com.ibm.ws.threading.internal.ExecutorServiceImpl$RunnableWrapper.run(ExecutorServiceImpl.java:239)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:836)
```

** Proposed fix **
Changing [SipMessage.java](https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/container/was/message/SipMessage.java) solved the problem. Inspiration was taken [here](https://groups.io/g/openliberty/message/308).

**Comment by annaguri** [comment]
@xjarkrof The fix for this issue will be available in the `21.0.0.1` release.

**Comment by xjarkrof** [comment]
> @xjarkrof The fix for this issue will be available in the `21.0.0.1` release.

Are there any release plans for 21.0.0.1 version ?

**Comment by annaguri** [comment]
@xjarkrof  end of January
